### PR TITLE
[WIP] Fix el6 box

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: config/base_boxes.yaml [hunk 1]
```diff
@@ -8,20 +8,16 @@ shells:
 
 boxes:
   centos6:
-    box_name:   'centos6'
+    box_name:   'centos/6'
```
Comment (stbenjam) [inline config/base_boxes.yaml:11]: By removing the URL's, and specifying the box name like this, we'll get the box from Vagrant atlas instead: https://atlas.hashicorp.com/boxes/search

The boxes we have here are ancient, using centos 6.4 and 7.1 - we should use newer (and need newer, right now failing for me on the 6.4 box)

Comment (ekohl) [reply config/base_boxes.yaml:11]: :+1:

**Comment by stbenjam** [comment]
On hold until the outcome of restoring Foreman's el6 nightlies, but I'll include the atlas change in a separate PR

**Comment by beav** [comment]
@stbenjam I tested this for el7 and it worked

**Pull request closed without merging**